app/views/shared/_comparison.html.erb

```diff
@@ -27,7 +27,7 @@
       <tr>
         <th></th>
         <% qhps.each do |qhp|%>
-          <th colspan="2" class="row-title" style="text-align: center; vertical-align:middle; border: none; padding: 0;"><%=number_to_currency(qhp[:total_employee_cost]) + " / month" %></th>
+          <th colspan="2" class="row-title" style="text-align: center; vertical-align:middle; border: none; padding: 0;"><%=number_to_currency(qhp[:total_employee_cost]).to_s + " / month" %></th>
```

Comment:
I don't want to silently fail and show 0 for a cost. Let's instead display an error.